Assets/AppCenter/Editor/BaseDependency.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using UnityEditor;
+using UnityEngine;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+public abstract class BaseDependency
+{
+    protected Type versionHandler;
```

Comment:
```suggestion
    protected Type VersionHandler { get; set; }
```

Best to use a property instead of a field for `protected` members. And with properties, the name usually starts with a capital letter. (Properties have `get` and `set` methods which can be customized, but fields act more like variables/constants.)

Reply:
You'll also need to change references to `versionHandler` to use the right casing